apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -291,7 +291,18 @@ protected TableMetadata mapTableResponseToTableMetadata(GetTableResponseBody res
             .dbName(responseBody.getDatabaseId())
             .tableName(responseBody.getTableId())
             .build();
-    String creator = getTable(metadata).getTableCreator();
+    GetTableResponseBody tableResponseBody = getTable(metadata);
+
+    if (tableResponseBody == null) {
+      log.error("Error while fetching metadata for table: {}", metadata);
+      return TableMetadata.builder()
+          .creator(null)
```

Comment:
good catch! we can skip those tables since we can't set proxy user

Reply:
I have updated to return null and skip including that is the results of getTables. PTAL.